src/nvim/ex_getln.c

```diff
@@ -4989,9 +5005,10 @@ static void expand_shellcmd(char_u *filepat, int *num_file, char_u ***file,
     if (e == NULL)
       e = s + STRLEN(s);
 
-    l = e - s;
-    if (l > MAXPATHL - 5)
+    l = (size_t)(e - s);
```

Comment:
PVS v1028

Reply:
not when the operands are pointers, I hope.